.github/workflows/ansible-lint.yml

```diff
@@ -1,11 +1,15 @@
-name: Run ansible-lint on the playbooks and roles
+# Run Ansible lint in playbooks and roles
+name: Run ansible-lint
 
 # Controls when the action will run.
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   pull_request:
     branches: [main]
-
+  push:
+    branches: [main]
+  schedule:
+    - cron: '0 */8 * * *'
```

Comment:
nevermind, I got it now:
```
https://github.com/openshift-psap/topsail/actions/workflows/ansible-lint.yml/badge.svg?event=schedule
```

Reply:
The idea of running this periodically is to catch new potential breakages coming from unpinned dependencies we have. For instance, if there are new lint rules included that are not addressed, this will work as an information radiator so we can jump in and fix the code before it affects unrelated PRs.

Reply:
you can see it here in my fork how the schedule job looks like in the readme https://github.com/ccamacho/topsail#topsail and in the rendered docs https://ccamacho.github.io/topsail/intro.html#topsail

Reply:
> The idea of running this periodically is to catch new potential breakages coming from unpinned dependencies we have. For instance, if there are new lint rules included that are not addressed, this will work as an information radiator so we can jump in and fix the code before it affects unrelated PRs.

very good point, I never really thought about indirect breakages from unpinned dependencies! 
we happened to have such an issue just [a few weeks ago](https://github.com/openshift-psap/matrix-benchmarking/pull/89/files), unfortunately in the case of Topsail, for cost sake, we don't want to run the meaningful tests periodically (=those running in OpenShift CI, not in GH Actions)